src/bin/commands/startup.rs

```diff
@@ -0,0 +1,22 @@
+use crate::config::default_ctants::{HOST_ADDRESS, PORT};
+use rusk::services::echoer::basic_proto::echoer_server::EchoerServer;
+use rusk::services::echoer::Rusk;
+use tonic::transport::Server;
+
+pub async fn startup(
+    host: Option<&str>,
+    port: Option<&str>,
```

Comment:
As discussed, remove `Option` here; you should pass directly the concrete value (or the default ones).

That would also remove the needs for `startup` to known the "default" values.

Reply:
Addressed.